pkg/fleetautoscalers/controller.go

```diff
@@ -122,28 +122,18 @@ func NewController(
 
 	autoscaler.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj interface{}) {
-			if runtime.FeatureEnabled(runtime.FeatureCustomFasSyncInterval) {
-				c.addFasThread(obj.(*autoscalingv1.FleetAutoscaler), true)
```

Comment:
I see it! 😄 you deleted the wrong side of the condition here when cleaning up the code.

So we want to keep the side that is true for `runtime.FeatureEnabled`

Reply:
I have substituted it with the correct statement. Both the `TestControllerEvents` function and `make test-go` have succeeded.